apps/web/src/components/milestone-logs/recruitment/Record.tsx

```diff
@@ -90,7 +81,7 @@ export const Record: React.FC<RecordProps> = ({ job }) => {
               <DetailsItem title='Recruiter Name' text={recruiter_name} />
               <DetailsItem
                 title='Date Job Was First Posted'
-                text={formatDate(job_post_date.toString())}
+                text={job_post_date && formatDate(job_post_date)}
```

Comment:
formatDate might handle null and undefined.

Reply:
you are correct, I will remove those checks